cli/src/command.rs

```diff
@@ -137,6 +138,13 @@ pub fn run() -> sc_cli::Result<()> {
         .execution
         .get_or_insert(ExecutionStrategy::Wasm);
 
+    // Set default base directory to `gear-node`.
+    cli.run
+        .base
+        .shared_params
+        .base_path
+        .get_or_insert_with(|| BasePath::from_project("", "", "gear-node").path().into());
```

Comment:
What's this path?

Reply:
- Changed default `--base-path` to `gear-node` instead of executable name (`gear[.exe]`) for backward compatibility.